tests/runtest.sh

```diff
@@ -1184,7 +1184,10 @@ then
     scriptPath=$(dirname $0)
     ${scriptPath}/setup-runtime-dependencies.sh --outputDir=$coreOverlayDir
 else
-    echo "Skip preparing for GC stress test. Dependent package is not supported on this architecture."
+    if [ ! "$ARCH" != "arm64" ]
```

Comment:
Looks like the logic is inverted.  I originally implemented this with a message saying it was not necessary on arm64.  When I removed it I must have inverted the logic.